CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
Leaving this in draft until https://github.com/awslabs/coldsnap/pull/58 is merged.